shared/settings/contacts-joined/index.native.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react'
+import * as Container from '../../util/container'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import {FollowButton, WaveButton} from './buttons'
+
+type ResolvedContactEntry = {
+  username: string
+  contactLabel: string
+}
+type Props = Container.RouteProps<{
+  people: Array<ResolvedContactEntry>
+}>
+
+const ContactsJoinedModal = (props: Props) => {
+  const people = Container.getRouteProps(props, 'people', [])
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+  const onClose = () => dispatch(nav.safeNavigateUpPayload())
+  return (
+    <Kb.Modal
+      header={{
+        leftButton: (
+          <Kb.Text type="BodyBigLink" onClick={onClose}>
+            Done
+          </Kb.Text>
+        ),
+      }}
+    >
+      <Kb.Text type="Body" style={styles.woot} center={true}>
+        Woot! Some of your contacts are already on Keybase.
+      </Kb.Text>
+      <Kb.ScrollView>
+        <Kb.Box2 direction="vertical" fullWidth={true}>
+          {people.map(({username, contactLabel}) => (
```

Comment:
I think this needs to be a real `List`

Reply:
What is the benefit of a `List`? / when should I use one generally?

Reply:
Whenever you have a potentially large # of items. Like if you have some view that is a static list of checkboxes, and you know there will only ever be like 5, no need for a `List`. But I have bad contacts hygiene with >700, without virtualization my phone is gonna struggle.